routes/products-routes.js

```diff
@@ -0,0 +1,56 @@
+const db = require("../models");
+
+module.exports = function(app){
+
+    // GET all products
+    app.get("/api/products", function(req,res){
+        db.Products.findAll({}).then(function(dbProducts){
+            res.json(dbProducts);
+        });
+    });
+
+    // GET product by ID
+    app.get("/api/products/:id", function(req, res) {
+        db.Products.findOne({
+        where: {
+            id: req.params.id
+        }
+        }).then(function(dbProduct) {
+        res.json(dbProduct);
+        });
+    });
+
+
+
+    // POST new products (may not need)
+    app.post("/api/products", function(req, res){
+        db.Products.create(req.body).then(function(dbProducts){
```

Comment:
Avoid passing in the whole body incase it's api changes in the future, always grab off exactly what you need.
Use es6 object initialization like I showed in the other comments.

{
  req.body.property1,
  req.body.property2,
  req.body.property3
}